extensions/metrics-cluster-feature/renderer.tsx

```diff
@@ -7,15 +7,13 @@ export default class ClusterMetricsFeatureExtension extends LensRendererExtensio
     {
       title: "Metrics Stack",
       components: {
-        Description: () => {
-          return (
-            <span>
-              Enable timeseries data visualization (Prometheus stack) for your cluster.
-              Install this only if you don't have existing Prometheus stack installed.
-              You can see preview of manifests <a href="https://github.com/lensapp/lens/tree/master/extensions/lens-metrics/resources" target="_blank">here</a>.
-            </span>
-          );
-        }
+        Description: () => (
+          <span>
+            Enable timeseries data visualization (Prometheus stack) for your cluster.
+            Install this only if you don&apos;t have existing Prometheus stack installed.
+              You can see preview of manifests <a href="https://github.com/lensapp/lens/tree/master/extensions/lens-metrics/resources" rel="noreferrer" target="_blank">here</a>.
+          </span>
+        )
```

Comment:
not important but can be without brackets `()`

Reply:
The style that seems to have been settled upon is that tsx "html" tags should generally be in the same column as their closing tag. I generally see this as easier to read. 

The one exception that I make is for `<> ... </>`.